Sources/Fisticuffs/Scheduler/Scheduler.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  Scheduler.swift
+//  Fisticuffs
+//
+//  Created by Eugene Kwong on 2021-04-10.
+//  Copyright © 2021 theScore. All rights reserved.
+//
+
+import Foundation
+
+public protocol Scheduler {
```

Comment:
Can you add some documentation for this? Also thoughts on having some convenience static properties to get common schedulers, i.e `.main` to get the main queue or `.default` to get an instance of the `DefaultScheduler` etc?

Reply:
Yep I'll be including that when I write out the docs.

Regarding the convenience static properties I'm not sure it makes sense to provide it here - seems like something that the app itself should define